tests/Unit/DataStructures/Tensor/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ set(LIBRARY_SOURCES
   Test_Slice.cpp
   Test_Tensor.cpp
   Test_TensorData.cpp
+  Test_TensorData.py
```

Comment:
I think you have to add python tests using something like 
```
# [example_add_pybindings_test]
spectre_add_python_bindings_test(
  "Unit.DataStructures.Python.DataVector"
  Test_DataVector.py
  "Unit;DataStructures;Python")
```
like in the `tests/Unit/DataStructures/CMakeLists.txt` file

Reply:
Delete this line please